tools/ci-build/runtime-versioner/src/command.rs

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use anyhow::{bail, Context, Result};
+use std::process::Command;
+
+/// Extension trait to make `Command` nicer to work with.
+pub trait CommandExt {
+    fn expect_success_output(&mut self, context: &str) -> Result<String>;
+
+    fn expect_statuses(&mut self, statuses: impl IntoIterator<Item = i32>) -> Result<i32>;
```

Comment:
from reading the code, I can see this is expecting that the status is one of a member of the list—didn't get that from name. Maybe a doc comment? Or `expect_status_one_of`